composer.json

```diff
@@ -66,6 +66,7 @@
         "symfony/twig-bundle": "^3.4 || ^4.0",
         "symfony/validator": "^3.3 || ^4.0",
         "symfony/web-profiler-bundle": "^3.3 || ^4.0",
+        "symfony/web-server-bundle": "^4.1",
```

Comment:
Can you allow lower versions? This constraint will prevent older versions of the deps to this bundle to be installed.